include/fem-dem/parameters_cfd_dem.h

```diff
@@ -116,6 +116,17 @@ namespace Parameters
     void
     parse_parameters(ParameterHandler &prm);
   };
+
+  struct CFDDEM_postprocessing
+  {
+    bool calculate_total_volume;
```

Comment:
I find "calculate total volume" a bit generic. What do you think about "compute volume phases"?

Reply:
I agree, I think that would makes more sense